packages/components/src/Components/NotAuthorized/NotAuthorized.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon403 from './Icon403';
+import {
+    Title,
+    Button
+} from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+
+const NotAuthorized = ({ serviceName, ...props }) => {
+    let history = useHistory();
```

Comment:
I don't know how to behave with hooks, we should check if every app is using newer version of React and if they are using it we should update the `react-router-dom` and `react` to newer version. But I'd rather use `withRouter` in here so we don't have any problems.